src/cmd/build.rs

```diff
@@ -293,6 +284,63 @@ fn optimize_wasm(crate_metadata: &CrateMetadata) -> Result<OptimizationResult> {
     })
 }
 
+/// Optimizes the Wasm supplied as `wasm` using the `binaryen-rs` dependency.
+///
+/// If successful, the optimized Wasm is returned as a `Vec<u8>`.
+#[cfg(feature = "wasm-opt-unavailable")]
```

Comment:
I would rename the crate feature to something "positive" such as `wasm-opt-installed` and then negate these conditions via: `#[cfg(not(feature = "wasm-opt-installed"))]`